src/ActionDialog/ActionDialog.js

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Stack
+} from "@mui/material";
+
+import CloseIcon from "@mui/icons-material/Close";
+import PropTypes from "prop-types";
+import { styled } from "@mui/material/styles";
+
+/**
+ * Horizontal alignment button group component to toggle between left/center/right/justify alignment
+ */
+export default function ActionDialog({
+  onCancelClick = () => {},
```

Comment:
It would be nicer if these props were in alphabetical order.

Reply:
Sorted